integration-tests/src/main/java/org/apache/druid/testing/utils/SqlQueryWithResults.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.testing.utils;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.druid.sql.http.SqlQuery;
+
+import java.util.List;
+import java.util.Map;
+
+public class SqlQueryWithResults
+{
+  private final SqlQuery sqlQuery;
+  private final List<Map<String, Object>> expectedResults;
+
+
+  @JsonCreator
+  @JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
What properties can be unknown and ignored?

Reply:
I think I was trying out something, this should not be required anymore.